collator/.env

```
@@ -0,0 +1,6 @@
+
+ADMIN_USER=admin
+ADMIN_PASSWORD=@dm1nPa55wd
+
+
```

We only need an empty line at the end. Also don't provide a default password. IMHO The user must set the password.

```suggestion
ADMIN_USER=admin
ADMIN_PASSWORD=
```